docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,134 @@
-  - diarizer.oracle_num_speakers = null (performing unknown speaker case)
-  - diarizer.speaker_embeddings.model_path = ``ecapa_tdnn`` (This model is trained on voxceleb dataset. ``Use this model for simialr non-telephonic speech datasets``)
-  - diarizer.speaker_embeddings.window_length_in_sec = 1.5
-  - diarizer.speaker_embeddings.shift_length_in_sec = 0.75 
 
-Input paths2audio_files, paths2rttm_files and oracle_vad_manifest by following steps as shown above
+Evaluation on benchmark dataset
+-------------------------------
 
-CallHome LDC97S42 (CH109)
--------------------------
+The following instructions can help you reproduce the expected diarization performance on two English dialogue datasets. The following results are evaluations based on 0.25 second collar without evaluating overlapped speech. The evaluation is based on oracle VAD results from RTTM files. Therefore, speaker error rate (SER) is equal to confusion error since oracle VAD has no miss detection or false alarm.
+
+AMi Meeting Corpus
+~~~~~~~~~~~~~~~~~~
+
+The following are the suggested parameters for reproducing the diarization performance for AMI test set.
+
+.. code-block:: bash
+
+  diarizer.manifest_filepath="/path/to/AMItest_input_manifest.json"
+  diarizer.oracle_num_speakers=null # Performing unknown speaker case
+  diarizer.oracle_vad=True # Use oracle VAD extracted from RTTM files.
+  diarizer.collar=0.25
+  diarizer.ignore_overlap=True 
+  diarizer.speaker_embeddings.model_path = ``titanet_large`` 
+  diarizer.speaker_embeddings.window_length_in_sec=[3,1.5,1.0,0.5] # Multiscale setting
+  diarizer.speaker_embeddings.shift_length_in_sec=[1.5,0.75,0.5,0.25] # Multiscale setting 
+  diarizer.speaker_embeddings.parameters.multiscale_weights=[0.4,0.3,0.2,0.1] # More weights on the longer scales
+
+This setup is expected to reproduce speaker error rate  of 1.19% on AMI test set.
+
+To evaluate the performance on AMI Meeting Corpus, the following instructions can help.
+  - Download AMI Meeting Corpus from `AMI website <https://groups.inf.ed.ac.uk/ami/corpus/>`_.
+  - Get the test set (whitelist) from `Pyannotate AMI testset whitelist <https://raw.githubusercontent.com/pyannote/pyannote-audio/master/tutorials/data_preparation/AMI/MixHeadset.test.lst>`_.
+  - The merged RTTM file for AMI testset can be downloaded from `Pyannotate AMI testset RTTM file <https://raw.githubusercontent.com/pyannote/pyannote-audio/master/tutorials/data_preparation/AMI/MixHeadset.test.rttm>`_. Note that this file should be split into individual rttm files. Download split rttm files for AMI testset from `AMI testset split RTTM files <https://raw.githubusercontent.com/tango4j/diarization_annotation/main/AMI_corpus/test/split_rttms.tar.gz>`_.
+  - Generate an input manifest file using ``<NeMo_git_root>/scripts/speaker_tasks/pathsfiles_to_manifest.py``
+
+
+CallHome American English Speech (CHAES), LDC97S42: 2-speaker subset (CH109)
```

Comment:
`: 2-speaker subset (CH109)`,  this better not be in heading as it's not looking good, and also since you mention it in the description part anyway.

Reply:
Removed